src/images/rainbowpath.svg

```diff
@@ -1,151 +1,139 @@
93.88 1625.36C2592.18 1629.73 2590.43 1634.11 2588.63 1638.48C2588.43 1638.98 2588.22 1639.48 2588.01 1639.99L2587 1642.12L2586.46 1643.41C2585.96 1644.61 2585.46 1645.81 2584.94 1647.01C2584.82 1647.32 2584.68 1647.63 2584.55 1647.94C2584.36 1648.4 2584.16 1648.86 2583.96 1649.32C2583.62 1650.13 2583.27 1650.94 2582.91 1651.75C2582.79 1652.05 2582.66 1652.35 2582.53 1652.65C2582.4 1652.95 2582.24 1653.32 2582.09 1653.65C2581.94 1653.98 2581.72 1654.51 2581.53 1654.94C2580.65 1656.94 2579.76 1658.94 2578.87 1660.99C2578.81 1661.12 2578.75 1661.26 2578.68 1661.39C2577.9 1663.17 2577.1 1664.95 2576.28 1666.73C2576.16 1667 2576.04 1667.27 2575.91 1667.54C2575.57 1668.3 2575.22 1669.07 2574.91 1669.83C2574.76 1670.17 2574.6 1670.51 2574.44 1670.83C2573.99 1671.83 2573.53 1672.83 2573.07 1673.83C2572.78 1674.47 2572.48 1675.1 2572.18 1675.74L2571.52 1677.15C2571.13 1677.98 2570.74 1678.82 2570.34 1679.65C2570.15 1680.06 2569.95 1680.48 2569.75 1680.89L2569.45 1681.5C2569.25 1681.94 2569.04 1682.39 2568.83 1682.83V1682.89C2568.66 1683.24 2568.5 1683.59 2568.32 1683.94C2568.03 1684.57 2567.73 1685.2 2567.42 1685.82L2567.18 1686.33C2566.83 1687.05 2566.48 1687.78 2566.12 1688.5C2565.58 1689.63 2565.03 1690.76 2564.46 1691.89C2564.39 1692.05 2564.31 1692.21 2564.23 1692.37C2564.15 1692.53 2564.02 1692.81 2563.91 1693.02C2563.53 1693.81 2563.14 1694.59 2562.75 1695.37L2561.3 1698.12C2560.14 1700.45 2558.97 1702.79 2557.79 1705.12C2556.71 1707.23 2555.63 1709.34 2554.54 1711.44C2594.06 1581.75 2590.41 1429.98 2546.46 1313.19H2600.27C2600.45 1313.56 2600.62 1313.93 2600.78 1314.31C2600.94 1314.69 2601.08 1314.98 2601.22 1315.31L2601.44 1315.8C2601.62 1316.17 2601.78 1316.54 2601.92 1316.91C2602.22 1317.59 2602.52 1318.27 2602.8 1318.91C2602.98 1319.33 2603.16 1319.74 2603.33 1320.16C2603.42 1320.36 2603.51 1320.55 2603.58 1320.75C2603.65 1320.89 2603.71 1321.03 2603.76 1321.18V1321.18C2603.83 1321.33 2603.89 1321.49 2603.95 1321.65C2604.34 1322.57 2604.72 1323.49 2605.1 1324.41C2605.77 1326.05 2606.44 1327.69 2607.1 1329.34C2607.51 1330.34 2607.91 1331.34 2608.3 1332.42C2608.63 1333.28 2608.96 1334.13 2609.3 1334.99C2609.47 1335.43 2609.64 1335.87 2609.8 1336.32C2610.09 1337.08 2610.37 1337.86 2610.65 1338.63C2611.41 1340.71 2612.15 1342.8 2612.87 1344.9C2613.23 1345.9 2613.59 1347.01 2613.94 1348.06C2614.19 1348.79 2614.43 1349.53 2614.67 1350.26C2614.67 1350.36 2614.74 1350.45 2614.77 1350.55L2615.16 1351.76L2615.22 1351.96L2616 1354.42C2616.62 1356.36 2617.21 1358.32 2617.78 1360.28C2618.05 1361.18 2618.31 1362.09 2618.57 1362.99C2618.99 1364.43 2619.39 1365.87 2619.78 1367.32C2620.05 1368.32 2620.32 1369.32 2620.58 1370.32C2620.76 1370.98 2620.94 1371.65 2621.1 1372.32C2621.26 1372.99 2621.37 1373.37 2621.5 1373.89C2621.72 1374.74 2621.93 1375.6 2622.14 1376.46L2622.58 1378.3C2622.86 1379.45 2623.13 1380.6 2623.38 1381.75C2623.58 1382.59 2623.76 1383.43 2623.94 1384.27C2624.12 1385.11 2624.32 1385.97 2624.49 1386.83C2624.71 1387.84 2624.92 1388.84 2625.1
```

Comment:
One side note: It seems like these linear gradient and filter ids would conflict between SVG files so it might be useful if we make them unique. 

I've been manually renaming them (e.g.: https://github.com/CruzHacks/cruzhacks-2021-website/pull/91/files#diff-5a5df358dc51c0b9b5722d062a56e0c8f8bb1dff0eb1cc4efd4d70b4b91f9cfcR8) but I think it is also possible to configure something to automatically attach a random id, though that may require more work.

Reply:
thanks for catching that @w-gao, this [NPM package](https://www.npmjs.com/package/react-svg-unique-id) gives us a neat wrapper for SVGs to dynamically create unique IDs. I'll update the branch with that drop-in component & hopefully it should resolve things.

@CruzHacks/2021-engineers we'll need to apply something like this to prevent ID collisions on SVG elements across everyone's components!